src/main/java/org/opensearch/flowframework/indices/FlowFrameworkIndicesHandler.java

```diff
@@ -301,6 +324,23 @@ public void putTemplateToGlobalContext(Template template, ActionListener<IndexRe
         }));
     }
 
+    /**
+     * Initializes master key index and EncryptorUtils
+     * @param listener action listener
+     */
+    public void initializeMasterKeyIndex(ActionListener<Boolean> listener) {
+        initMasterKeyIndexIfAbsent(ActionListener.wrap(indexCreated -> {
+            if (!indexCreated) {
+                listener.onFailure(new FlowFrameworkException("No response to create global_context index", INTERNAL_SERVER_ERROR));
+                return;
+            }
+            encryptorUtils.initializeMasterKey(listener);
```

Comment:
just wondering did you see if there was any special settings ml-commons is adding to there system index with the key